tools/topic_tools/scripts/transform

```diff
@@ -75,10 +75,15 @@ class TopicOp:
 
         self.expression = args.expression
 
+        if len(args.input) > 0 and args.input[0] != '/':
+            input_topic_in_ns = rospy.get_namespace() + args.input
+        else:
+            input_topic_in_ns = args.input
```

Comment:
This could be simplified to the following (assuming `args.topic` can't be empty):

```
input_topic = args.input
if not input_topic.startswith('/'):
    input_topic = rospy.get_namespace() + input_topic